File: BAEKJOON/Python/2000/2500/2566.py
"""
Title : 최댓값
Link : https://www.acmicpc.net/problem/2566
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    column, row = 0, 0
    max_value = 0
    for i in range(9):
        arr = list(map(int, input().strip().split()))
        if (max_in_row := max(arr)) > max_value:
            max_value = max_in_row
            column, row = i, arr.index(max_value)
    print(f"{max_value}\n{column + 1} {row + 1}")

File: BAEKJOON/Kotlin/2000/2500/2566.kt
// Title : 최댓값
// link : https://www.acmicpc.net/problem/2566

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() =
    with(BufferedReader(InputStreamReader(System.`in`))) {
        var column = 0
        var row = 0
        var max = -1
        for (i in 1..9) {
            val arr = readLine().split(" ").map { it.toInt() }
            for (j in 1..9) {
                if (arr[j - 1] > max) {
                    max = arr[j - 1]
                    column = i
                    row = j
                }
            }
        }
        println(max)
        print("$column $row")
    }
